src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/Constants.java

```diff
@@ -20,6 +20,7 @@ private Constants() {
     // Request attribute names
     public static final String TRANSACTION_KEY = "transaction";
 
-    // Filings
+    public static final String OVERSEAS_ENTITY_ID_KEY = "overseas_entity_id";
```

Comment:
This is used to reference a path variable like the TRANSACTION_ID_KEY so please move up, just under that constant on line 12.

Reply:
@mattch1 I think you missed this? No response to my comment so not sure. Could you please action.

Reply:
Oh I see, you moved the wrong lines. TRANSACTION_KEY constant and comment should remain where it is but the new OVERSEAS_ENTITY_ID_KEY constant needs to move up to just below line 12, as it refers to a URI path attribute.

Reply:
@mattch1 this still needs attention.